### PR TITLE
Add plan review for Reactive Streams Operators 3.0

### PR DESCRIPTION
**Pull request by Emily-Jiang**

### CODE REVIEW

File: ReactiveStreamsOperators/3.0/README.adoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: "MicroProfile Reactive Streams Operators 3.0 (under development)"
+date: 2022-07-28
+summary: "Release for MicroProfile Reactive Streams Operators 3.0"
```
Comment (ederks85) [inline ReactiveStreamsOperators/3.0/README.adoc:4]: In the Plan Review for OpenAPI 3.1, the description is this "Release for MicroProfile 6.0". This doesn't seem to be consistent; does the description have to relate to a/the release, to the spec release itself or is it OK to just have anything here?

Comment (Emily-Jiang) [reply ReactiveStreamsOperators/3.0/README.adoc:4]: This Reactive Streams Operators is standalone, so it is not part of MP 6.0

Comment (ederks85) [reply ReactiveStreamsOperators/3.0/README.adoc:4]: Right, thanks for pointing that out.

**Review by ederks85 — APPROVED** [review]

**Comment by JanWesterkamp-iJUG** [comment]
@Emily-Jiang in the voting result list Heiko W. Rupp as iJUG Alternate is missing - can you please add him at least with the voting resutls?
Thanks!

**Comment by Emily-Jiang** [comment]
> @Emily-Jiang in the voting result list Heiko W. Rupp as iJUG Alternate is missing - can you please add him at least with the voting resutls? Thanks!

@JanWesterkamp-iJUG have you informed Eclipes Foundation? @jclingan fyi please update your working group minutes template with this info.

**Comment by JanWesterkamp-iJUG** [comment]
> > @Emily-Jiang in the voting result list Heiko W. Rupp as iJUG Alternate is missing - can you please add him at least with the voting resutls? Thanks!
> 
> @JanWesterkamp-iJUG have you informed Eclipes Foundation? @jclingan fyi please update your working group minutes template with this info.

@Emily-Jiang yes, the official process with EF (announcing on the WG mailing list) was done and the Excel sheed got updated:
https://docs.google.com/spreadsheets/d/1QflmI_5NWLEGDGImibvMx5iSFUZisbnHjMvvOIcLRy0/edit#gid=0

**Pull request merged**